doc/developer.md

```diff
@@ -12,33 +12,28 @@ git clone https://github.com/sourmash-bio/sourmash.git
 ```
 sourmash runs under Python 3.10 and later.
 
-We recommend using `conda` or `Nix` for setting up an environment for developing
+We recommend using `pixi` or `Nix` for setting up an environment for developing
 new features, running tests and code quality checks.
 Here are some suggestions on how to set them up (note: you only need one =])
 
-### Using mamba (conda alternative)
+::::{tab-set}
```

Comment:
How the tabs look like: https://sourmash--2990.org.readthedocs.build/en/2990/developer.html
![image](https://github.com/sourmash-bio/sourmash/assets/6642/9790ce48-5bb3-4603-a023-10f60b5d813a)

Reply:
I love it!